src/trafficcontroller/dopplerproxy/doppler_proxy.go

```diff
@@ -137,13 +137,24 @@ func (proxy *Proxy) serveAppLogs(paths []string, writer http.ResponseWriter, req
 		return
 	}
 
-	authorized, err := proxy.logAuthorize(authToken, appId, proxy.logger)
+	status, _ := proxy.logAuthorize(authToken, appId, proxy.logger)
```

Comment:
Why is the error being ignored? Messages like the `NO_AUTH_TOKEN_PROVIDED_ERROR_MESSAGE` will be lost here. I can see that in the case of a `403` status code, the error message returned would be `Forbidden` which I guess we want to obfuscate and return `Not Found`.

Maybe we should just return a status code?